crates/icp/src/identity/key.rs

```diff
@@ -76,26 +81,28 @@ pub fn load_identity(
     match identity {
         IdentitySpec::Pem {
             format, algorithm, ..
-        } => load_pem_identity(
-            dir,           // dir
-            name,          // name
-            format,        // format
-            algorithm,     // algorithm
-            password_func, // password_func
-        ),
+        } => {
+            load_pem_identity(
+                dirs,          // dir
+                name,          // name
+                format,        // format
+                algorithm,     // algorithm
+                password_func, // password_func
+            )
```

Comment:
```suggestion
            load_pem_identity(
                dirs,
                name,
                format,
                algorithm,
                password_func,
            )
```

feels a bit redundant